swagger.json

```diff
@@ -1821,6 +1821,57 @@
             }
           }
         },
+        "/fleet/vehicles/{vehicle_id}/stats": {
```

Comment:
out of curiosity, is there an interface or generator to create this json? or do we have to hand write it?

Reply:
We have to hand write it at the moment.